tests/test_indexer.py

```diff
@@ -1426,7 +1449,7 @@ def create_index_data(blobstore, bucket_name, bundle_key, manifest,
     index = dict(state="new", manifest=manifest, uuid=BundleFQID.from_key(bundle_key).uuid)
     files_info = manifest['files']
     excluded_file = [file.replace('_', '.') for file in excluded_files]
-    index_files = {}
+    index_files: typing.Dict = {}
```

Comment:
See above.

Reply:
?

Reply:
Never mind. It's a bug in mypy, I guess. This shouldn't be necessary.